components/share-modal/share-modal.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "variables.scss";
+.ShareModalCont {
+  padding: 1rem;
+  width: 240px;
+  height: 289px;
+  background: #1a1b1d 0% 0% no-repeat padding-box;
+  box-shadow: 0px 3px 6px #00000029;
+  border-radius: 4px;
+  opacity: 1;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  .title .share {
+    color: #d8d8d8;
+    opacity: 1;
+    text-align: left;
+    font-size: 18px;
```

Comment:
font-size are already a variable in the _variables.scss file. Remember each team = 10px so in this case will be the variable with the 1.8rem value.

The same goes for all the font-size use in this file.